arkouda/pdarraysetops.py

```diff
@@ -275,7 +275,8 @@ def concatenate(arrays: Sequence[Union[pdarray, Strings, 'Categorical', ]],  # t
         raise TypeError('arrays must be an array of pdarray or Strings objects')
 
 
-def multiarray_setop_validation(pda1: List[pdarray], pda2: List[pdarray]):
+def multiarray_setop_validation(pda1: Union[List[pdarray], Tuple[pdarray, ...]],
```

Comment:
Sorry for just noticing this, but should we make these `Union[List[groupable], Tuple[groupable]]`? I'm 95% sure all the logic will work with Strings as well as pdarrays, but maybe we should leave that for a separate PR, because it would require additional tests.

Reply:
No problem. I will add an Issue to update the typing and add additional testing.

Reply:
Issue #1280 has been added to track this.